circus-train-tool-parent/pom.xml

```diff
@@ -80,6 +80,12 @@
     <dependency>
       <groupId>com.hotels</groupId>
       <artifactId>beeju</artifactId>
+      <exclusions>
```

Comment:
Could you rather upgrade to the latest version of BeeJU as this change has been applied there (see BeeJU's CHANGELOG for the 2.0.0 release) so hopefully this dependency no longer comes through transitively?